satpy/etc/readers/atms_l1b_nc.yaml

```
@@ -0,0 +1,394 @@
+reader:
+  name: atms_l1b_nc
+  short_name: ATMS L1B NetCDF4
+  long_name: JPSS ATMS L1B (NetCDF4)
+  description: >
+    Reader for JPSS Advanced Technology Microwave Sounder Level 1B files in NetCDF4.
+  status: Beta
+  sensors: [atms]
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
```

```suggestion
  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
  supports_fsspec: false
```

This additional information was added recently and is described here: https://satpy.readthedocs.io/en/latest/dev_guide/custom_reader.html#the-reader-section

---

thanks for letting me know.